packages/figma/src/lib/TokensData.ts

```diff
@@ -0,0 +1,69 @@
+export type TokenPrimitiveValue = string | number
+
+export type TokenValue = {
+  value: TokenPrimitiveValue
+  description?: string
+}
+
+export type TokensGroup = { [key: string]: TokenPrimitiveValue | TokenValue | TokensGroup }
```

Comment:
В ts есть встроенный тип для словарей:

```suggestion
export type TokensGroup = Record<string, TokenPrimitiveValue | TokenValue | TokensGroup>
```